### PR TITLE
Update MCAD crd descriptions

### LINKED ISSUE #57: Update MCAD CRD to remove `TODO:` Descriptions
We have a number of TODO descriptions in the mcad custom resource. These should be updated to include "real" descriptions.

https://github.com/project-codeflare/codeflare-operator/blob/main/api/v1alpha1/mcad_types.go

### PR DESCRIPTION
**Pull request by tedhtchang**
Closes #57

### CODE REVIEW

File: api/v1alpha1/mcad_types.go [hunk 1]
```diff
@@ -38,13 +38,13 @@ type MCADSpec struct {
...
-	// AgentConfigs TODO: Add details
+	// AgentConfigs determine paths to agent config file:deploymentName separted by commas(,).
 	AgentConfigs string `json:"agentConfigs,omitempty"`
 
-	// QuotaRestURL TODO: Add details
+	// QuotaRestURL determines URL for Rest quota management.
 	QuotaRestURL string `json:"quotaRestURL,omitempty"`
 
-	// PodCreationTimeout TODO: Add details and confirm values
+	// PodCreationTimeout determins timeout in milliseconds for pods to be created after dispatching job.
```
Comment (KPostOffice) [inline api/v1alpha1/mcad_types.go:47]:
```suggestion
	// PodCreationTimeout determines timeout in milliseconds for pods to be created after dispatching job.
```

**Review by KPostOffice — COMMENTED** [review]
One small comment

File: api/v1alpha1/mcad_types.go (hunk 1 above)
Comment (tedhtchang) [reply api/v1alpha1/mcad_types.go]: Thanks @KPostOffice !

File: config/crd/bases/codeflare.codeflare.dev_mcads.yaml [hunk 2]
```diff
@@ -102,15 +103,16 @@ spec:
                 type: boolean
               podCreationTimeout:
                 default: -1
-                description: 'PodCreationTimeout TODO: Add details and confirm values'
+                description: PodCreationTimeout determins timeout in milliseconds
```
Comment (asm582) [inline config/crd/bases/codeflare.codeflare.dev_mcads.yaml:106]: Fix typo

Comment (tedhtchang) [reply config/crd/bases/codeflare.codeflare.dev_mcads.yaml]: Thanks!

**Review by anishasthana — APPROVED** [review]
/lgtm

**Pull request merged**